github/orgs_teams.go

```diff
@@ -46,17 +46,46 @@ func (t Team) String() string {
 
 // Invitation represents a team member's inviation status
 type Invitation struct {
-	ID        *int       `json:"id,omitempty"`
-	Login     *string    `json:"login,omitempty"`
-	Email     *string    `json:"email,omitempty"`
-	Role      *string    `json:"role,omitempty"`
-	CreatedAt *time.Time `json:"created_at,omitempty"`
+	ID    *int    `json:"id,omitempty"`
+	Login *string `json:"login,omitempty"`
+	Email *string `json:"email,omitempty"`
+	Role  *string `json:"role,omitempty"`
+	//golang documentation states
+	//Programs using times should typically store and pass them as values, not pointers.
+	//That is, time variables and struct fields should be of type time.Time, not *time.Time
+	CreatedAt time.Time `json:"created_at,omitempty"`
+	Inviter   *Inviter  `json:"inviter,omitempty"`
 }
 
 func (i Invitation) String() string {
 	return Stringify(i)
 }
 
+// Inviter represents the entity introduced in team and org invitations
```

Comment:
A period is missing at the end of this sentence.

Also, you can write the full word "organization", there's no need for a short form here.